src/core/format/RawAddress.ts

```diff
@@ -105,10 +106,11 @@ export class RawAddress {
     /**
      * Determines the validity of a decoded address.
      * @param {Uint8Array} decoded The decoded address.
-     * @param {SignSchema} signSchema The Sign Schema. (KECCAK_REVERSED_KEY / SHA3)
+     * @param {number} networkIdentifier The network identifier.
```

Comment:
what is network identifier ? Is it a `networkType` ? or maybe do we need an added model NetworkIdentifier ?

Reply:
I think's it's a typo. should read NetworkType